packages/shared/src/components/ProfilePicture.tsx

```diff
@@ -120,4 +120,47 @@ function ProfilePictureComponent(
   );
 }
 
-export const ProfilePicture = forwardRef(ProfilePictureComponent);
+const Group: React.FC<{
+  children:
+    | React.ReactElement<ProfilePictureProps>[]
+    | React.ReactElement<ProfilePictureProps>;
+  limit?: number;
+  total?: number;
+}> = ({ children, limit = 3, total }) => {
+  const childrenMap = React.Children.toArray(children).slice(0, limit);
+  const remainingCount = total ? total - childrenMap.length : 0;
```

Comment:
This will likely be a set data you already receive calculated

Reply:
Response will be already limited amount of sources and total number, so I think this is needed. I think we can leave `limit` as well, we might want to manually limit even more in some cases.